src/main/java/mods/railcraft/common/blocks/machine/gamma/TileDispenserCart.java

```diff
@@ -125,7 +127,7 @@ public void onPulse() {
                             float rx = MiscTools.getRand().nextFloat() * 0.8F + 0.1F;
                             float ry = MiscTools.getRand().nextFloat() * 0.8F + 0.1F;
                             float rz = MiscTools.getRand().nextFloat() * 0.8F + 0.1F;
-                            EntityItem item = new EntityItem(worldObj, x + rx, y + ry, z + rz, cartStack);
+                            Entity item = cartStack.getItem().createEntity(worldObj, new EntityItem(worldObj, x, y, z, cartStack), cartStack);
```

Comment:
You haven't tested this have you? The default implementation of Item.createEntity() just returns null.